app/packs/stylesheets/decidim/participatory_documents/pdf/defaults.scss

```
@@ -0,0 +1,7 @@
+:root {
+  --box-text-color: #fff;
+  --box-color: rgb(30, 152, 215);
+  --box-color-rgba: rgba(30, 152, 215, .12);
+  --notifications-color-rgba: rgba(30, 152, 215, .6);
+  --suggestions-modal-color-as-hsla: hsla(213, 48%, 90%, .95);
```

```suggestion
  --suggestions-modal-color-hsla: hsla(213, 48%, 90%, .95);
```
We can simplify the name a bit